src/core_codemods/process_creation_sandbox.py

```diff
@@ -5,11 +5,21 @@
 
 class ProcessSandbox(SemgrepCodemod):
     NAME = "sandbox-process-creation"
-    REVIEW_GUIDANCE = ReviewGuidance.MERGE_AFTER_CURSORY_REVIEW
-    SUMMARY = "Sandbox process creation"
+    REVIEW_GUIDANCE = ReviewGuidance.MERGE_WITHOUT_REVIEW
```

Comment:
I know you're probably just matching whatever is currently in the docs but I don't think any of our codemods that add dependencies should be `MERGE_WITHOUT_REVIEW` right now.

Reply:
right I was matching that. I can switch it to merge after cursory review then